packages/suite/src/components/suite/DeviceDisplay/parseTextToPagesAndLines.ts

```diff
@@ -52,10 +53,26 @@ export const parseTextToPagesAndLines = ({
     deviceModel,
     text,
 }: ParseTextToLinesParams): ParseTextToLinesResult => {
+    const getCharsPerRow = (): number => {
+        // T1B1 has varying row length depending on address type.
+        if (deviceModel === DeviceModelInternal.T1B1) {
+            switch (text.length) {
```

Comment:
Is the text length enough to assume the type of the address? What about other coins?

Reply:
It possibly doesn't matter whether the text is an address or not and the device always breaks lines like this. Not sure. tested with Litecoin and it works, too.